src/main/java/org/opentripplanner/updater/bike_rental/datasources/UIPBikeRentalDataSource.java

```diff
@@ -8,13 +8,18 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 
 class UIPBikeRentalDataSource extends GenericJsonBikeRentalDataSource {
 
     private String baseURL = null;
 
     public UIPBikeRentalDataSource(BikeRentalDataSourceParameters config) {
-        super(config, "stations", "Client-Identifier", config.getApiKey());
+        super(
+            config,
+            "stations",
+            config.getApiKey() != null ? Map.of("Client-Identifier", config.getApiKey()) : Map.of()
```

Comment:
Is it now possible to remove the Entur specific ApiKey? An replace it just by adding config in our deployment ?

Reply:
That would be possible, but we are not using this updater anymore, and I think it might be removed by @leonardehrenfried  soon?